barf.go

```diff
@@ -1,4 +1,6 @@
-/* package barf
+/*
+	package barf
```

Comment:
For the formatting here....I agree is has to come down but should be shifted to the far left...basically, the starting of the line.

Reply:
Hm, `gofmt` going funky on my end it seems. Have to fix that somehow, will commit the right thing with the next one.